src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result -= 50;
+  } else if (days >= 3) {
+    result -= 20;
+  }
```

Comment:
The numbers `40`, `50`, `20`, `7`, and `3` are hard-coded values. It's a code quality issue. It's a good practice to avoid using literal numbers directly in expressions, as it makes the code harder to understand and maintain. Instead, declare them as named constants at the top of your function. This makes the code self-explanatory.